mflowgen/common/custom-power-sram/outputs/power-strategy-dualmesh.tcl

```diff
@@ -0,0 +1 @@
+../../power-scripts/srams-power-strategy-dualmesh.tcl
```

Comment:
Same with this?

Reply:
Shouldn't this also be called something like leaf-with-srams-power-strategy?

Reply:
I think combining the two leaves is a good idea .. it should be possible to skip the M5 power straps if there are no SRAMs in order to avoid the errors, right?

Reply:
I think if you do `get_cells -quiet` it will suppress the errors.

Reply:
Oh nice. I'll try it with `-quiet` and combine them